config/database.yml

```diff
@@ -30,7 +30,7 @@ review:
 
 performance:
   <<: *default
-  database: 'early_careers_framework_performance'
```

Comment:
not sure about updating this as its a special performance env, just wanted to double check?

Reply:
yeah you're right, it doesn't look like its deployed to Azure; I'll revert this